core/conversion/converters/Weights.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "core/util/prelude.h"
+#include "core/conversion/conversionctx/ConversionCtx.h"
+
+namespace trtorch {
+namespace core {    
+namespace conversion {
+namespace converters {
+
+struct Weights {
+    //TODO: Rebuild this in a way that makes sense for more than just conv2/3D and linear
+    nvinfer1::Weights data;
+    nvinfer1::Dims kernel_shape;
+    nvinfer1::Dims shape;
+    int64_t num_input_maps;
+    int64_t num_output_maps;
+
+    Weights();
+    Weights(ConversionCtx* ctx, at::Tensor t);
+    Weights(ConversionCtx* ctx, float val);
+    Weights(ConversionCtx* ctx, int32_t val);
+    friend std::ostream& operator<<(std::ostream& os, const Weights& w);
+};
+
+inline nvinfer1::ITensor* tensor_to_const(ConversionCtx* ctx, at::Tensor t) {

```

Comment:
This and its uses can be replaced now I think since we have `ITensorOrFreeze`

Reply:
This function doesn't seem to have any uses, so I'll remove it.